adapters/sidekiq/lib/opentelemetry/adapters/sidekiq/middlewares/client/tracer_middleware.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module Sidekiq
+      module Middlewares
+        module Client
+          class TracerMiddleware
+            def call(worker_class, job, _queue, _redis_pool)
+              tracer.in_span(
+                worker_class,
+                attributes: {
+                  jid: job['jid'],
```

Comment:
Might be nicer to call this `job_id`.

Also, attribute keys need to be strings. We don't support symbols. We don't validate this at the moment, but #207 may add some of that back.